Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptSearch.java

```diff
@@ -123,10 +127,15 @@ private void initBinder(WebDataBinder binder) {
             totalNumberOfRecords = manager.getTotalNumberOfRecordsForSearch(searchFields, operator);
             searchResults = manager.searchForConceptByPageNumberAndFieldMap(searchFields, operator,
                     page, numberOfRecordsPerPage);
-        } catch (LuceneException | IllegalAccessException ex) {
+        } catch (LuceneException ex) {
+            logger.error("Lucene Exception", ex);
             return new ResponseEntity<String>(ex.getMessage(), HttpStatus.INTERNAL_SERVER_ERROR);
-        } catch (IndexerRunningException e) {
-            return new ResponseEntity<String>(e.getMessage(), HttpStatus.SERVICE_UNAVAILABLE);
+        } catch (IllegalAccessException iae) {
+            logger.error("Illegal access exception", iae);
+            return new ResponseEntity<String>(iae.getMessage(), HttpStatus.BAD_REQUEST);
```

Comment:
That has not been thrown. I added it without noticing that. Removed that catch statements.